src/event_handler.rs

```diff
@@ -101,9 +102,19 @@ impl EventHandler {
                     continue;
                 }
             }
+            //checking if there's a "fake" key version of a relative event, (scancodes equal to and over 59974 are only "fake" custom events)
+            //and also if it's the same "key" and value as the one that came in.
+            if key.code() >= 59974 && (key.code(), value) == (event.code(), event.value()) {
```

Comment:
Is there no way to avoid this magic number `59974` using something in the evdev crate? I think you could at least make it a const instead of writing that everywhere.